projects/storefrontlib/src/cms-components/product/product-images/product-images.component.ts

```diff
@@ -23,13 +23,18 @@ export class ProductImagesComponent {
   );
 
   thumbs$: Observable<any[]> = this.product$.pipe(
-    map(product => this.createThumbs(product))
+    map(product => this.createThumbs(product)),
+    tap(thumbs => {
+      if (thumbs.length === 0) this.isEmpty = true;
```

Comment:
also need to add `else {this.isEmpty = false}` or `this.isEmpty = thumbs.length === 0`